support/xspec-dev/testing.md

```diff
@@ -0,0 +1,24 @@
+# Testing
+
+This work in support of XSpec is in need of more rigorous testing.
+
+Interactive testing has been performed with all scripts.
+
+But requirements and specifications are loose, having been driven up to this point by functional requirements faced by the metaschema-xslt project - for a flexible, performant runtime that works independently of licensed software, with developer-friendly controls from the command line and inside CI/CD. Specifically, this means a number of issues, including testing, that bear on full integration back into XSpec have been set aside for the present.
+
+Some experience in application will also be helpful in prioritizing the testing work to be done.
+
+Meanwhile, some interesting questions, both practical and theoretical, are posed by the need to test these capabilities, especially outside particular deployments.
+
+For which capabilities can we write XSpec or meaningfully test and confirm (expose and document) in other ways, including
+
+  - `xsl:message` behavior including terminate="yes"
```

Comment:
In case you're not already aware of it, XSpec can test terminating messages
https://github.com/xspec/xspec/wiki/Testing-Dynamic-Errors

I don't know a way to make XSpec test non-terminating `xsl:message` instructions, though. That's why I gave the xslt3-functions message handler an [option](https://github.com/usnistgov/xslt3-functions/blob/0f41f56caeeefb39c090a1f3e587dd95b3136088/message-handler/message-handler.xsl#L16) to emit a processing instruction instead of `xsl:message`. The idea is to use the message in production and the PI (a "mock" message) when testing.